src/main/java/Deadline.java

```diff
@@ -0,0 +1,23 @@
+package duke;
+
+/**
+ * Represents the Deadline Task object.
+ */
+public class Deadline extends Task {
+
+    /**
+     * Initializes the Deadline object
+     * @param description string which describes the task.
+     */
+    public Deadline(String description) {
+        super(description);
+    }
+
+    /**
+     * Returns a formatted string representing the Deadline task.
+     * @return String
+     */
+    public String toString() {
+        return String.format("[D][%s] %s", getStatusIcon(), description);
+    }
```

Comment:
Maybe could consider adding the `@Override` annotation above all the `toString` methods? May not be necessary (coding standard does not talk about it) but I would think that it is recommended.
```suggestion
    @Override
    public String toString() {
        return String.format("[D][%s] %s", getStatusIcon(), description);
    }
```